frontend/public/components/sidebars/resource-sidebar.jsx

```diff
@@ -28,12 +28,12 @@ export class ResourceSidebarWrapper extends React.Component {
   }
 
   render() {
-    const {style, label, linkLabel, children} = this.props;
+    const {style, label, linkLabel, children, isCreateMode} = this.props;
     const {height} = style;
     const {showSidebar} = this.state;
 
     if (!showSidebar) {
-      return <div className="co-p-has-sidebar__sidebar--hidden hidden-sm hidden-xs">
+      return <div className={`co-p-has-sidebar__sidebar--hidden hidden-sm hidden-xs ${isCreateMode && 'co-p-has-sidebar__new-mode'}`}>
```

Comment:
It's a shame we have to pass a prop all the way through just to move the link 10px higher. I think we should work with @sg00dwin to find a cleaner way to position the link and try to avoid adding the prop.